cmd/producer/main.go

```diff
@@ -33,14 +34,16 @@ func run() error {
 	for {
 		time.Sleep(time.Second * 2)
 		t := time.Now().String()
+		correlationId := uuid.NewString()
```

Comment:
I think the user of this library should generate this. Right now you would not be able to find the inspector TTL handler logs would using the correlationId attribute. The goal is that a single actions logs are searchable using the correlationId.

Reply:
my bad, I have mistaken this file with the actual library producer 😅